core/src/dird/volume_key.cc

```
@@ -0,0 +1,88 @@
+        T_("Failed to generate new encryption passphrase for Volume %s.\n"),
+        mr->VolumeName);
+    return false;
+  }
+
+  // See if we need to wrap the passphrase.
+  if (me->keyencrkey.value) {
+    char* wrapped_passphrase;
+
+    length = DEFAULT_PASSPHRASE_LENGTH + 8;
+    wrapped_passphrase = (char*)malloc(length);
+    memset(wrapped_passphrase, 0, length);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)